qtgqlcodegen/templates/operation.jinja.hpp

```diff
@@ -28,7 +34,7 @@ void update_👉 t.name 👈(👉 t.concrete.member_type_arg 👈 inst, const QJ
 
 // ------------ Narrowed Interfaces ------------
 {% for t in context.operation.interfaces -%}
-class 👉 t.name 👈: public 👉 context.qtgql_types.ObjectTypeABC.name 👈{
+class QTGQL_EXPORT_FOR_UNIT_TESTS 👉 t.name 👈: public 👉 context.qtgql_types.ObjectTypeABC.name 👈{
```

Comment:
Is that export thing is required for every class? because there are many more classes in `schema.jinja.hpp`

Reply:
actually it is good to put the definition of the export somewhere in a common global file to be imported instead of defining this in every header file like here: https://github.com/KDAB/KDDockWidgets/blob/1a243ac41e207b716b9f6536a47c575d9ede4eea/src/docks_export.h#L30

but yes we have to put this for all classes that we want to use from this library in test executable

Reply:
Hmm AFAIK only input object (that are defined in `schema.jinja.hpp`) are used by external user.

Reply:
are you using those schema classes in test executable too?
I didn't encountered any error while testing other test cases. it was just the classes and methods in `operation.jinja.hpp`

Reply:
this seems to be an struct but if we use classes in schema.jinja.hpp too, we have to put this export thing also on their class definition too